src/parameters/all_parameters.cpp

```diff
@@ -110,9 +122,12 @@ void AllParameters::declare_parameters (dealii::ParameterHandler &prm)
                       " shock_1d | "
                       " euler_naca0012 | "
                       " reduced_order | "
+//                      " burgers_rewienski_snapshot |"
```

Comment:
why is this being commented out?

Reply:
fixed by deleting. Was from an older version of the code.